### PR TITLE
Fixes#412: user can now add background image to message section

### PR DESCRIPTION
**Pull request by madhavrathi**
Fixes issue #412 

Changes: Now user can choose background for message section as well.

Demo Link: http://susimadhav.surge.sh/

Screenshots for the change: 

aajkaissuefix
@rishiraj824 @uday96 @isuruAb @daminisatya Please Review
Thanks

### CODE REVIEW

File: src/components/ChatApp/MessageSection/MessageSection.react.js [hunk 1]
```diff
@@ -546,7 +565,7 @@ class MessageSection extends Component {
                 <ul
                   className='message-list'
                   ref={(c) => { this.messageList = c; }}
-                  style={{background:pane}}>
+                  style={messageBackgroundStyles}>
```
Comment (rishiraj824) [inline src/components/ChatApp/MessageSection/MessageSection.react.js]: Let's have the `background-repeat:none` property as well to the styles. @madhavrathi

**Comment by madhavrathi** [comment]
@rishiraj824 Please Review. @isuruAb I am able to change image just fine, can you try again with a different image. For ex. try this url https://image.freepik.com/free-vector/abstract-blue-background_1048-1511.jpg
Thanks

**Comment by rishiraj824** [comment]
@uday96 Probably, this will only be a one time use feature because currently, the server is not going to store any file of a user. If I am not wrong the Android Client does it but it happens locally only on the App https://github.com/fossasia/susi_android/pull/504

**Comment by rishiraj824** [comment]
screenshot from 2017-07-08 14 32 47
@madhavrathi What I meant by adding a `background-repeat : none` was that the background should not repeat itself over the area and cover the whole area. So maybe https://www.w3schools.com/cssref/playit.asp?filename=playcss_background-size&preval=cover
this link can help

**Comment by madhavrathi** [comment]
aajkaissuefix

@rishiraj824 It all looks good now. Please Review http://susimadhav.surge.sh/

**Pull request merged**